src/sdg/api/serializers/producten.py

```diff
@@ -25,27 +25,74 @@
 class LocalizedProductSerializer(serializers.ModelSerializer):
     """Serializer for the localized version of a product."""
 
-    verwijzing_links = LabeledUrlListField()
+    verwijzing_links = LabeledUrlListField(
+        help_text="Dit zijn de verwijzing links voor burgers en ondernemers naar relevante organisatie informatie."
+    )
 
     class Meta:
         model = LocalizedProduct
         fields = (
             "taal",
+            "product_titel_decentraal",
             "specifieke_tekst",
+            "verwijzing_links",
+            "procedure_beschrijving",
             "bewijs",
+            "vereisten",
             "bezwaar_en_beroep",
-            "decentrale_procedure_link",
             "kosten_en_betaalmethoden",
-            "procedure_beschrijving",
-            "product_titel_decentraal",
             "uiterste_termijn",
-            "vereisten",
-            "verwijzing_links",
             "wtd_bij_geen_reactie",
-            "datum_wijziging",
+            "decentrale_procedure_link",
             "product_aanwezig_toelichting",
             "product_valt_onder_toelichting",
+            "datum_wijziging",
         )
+        extra_kwargs = {
+            "taal": {
+                "help_text": """Dit is de taal van de onderstaande gegevens, het is de bedoeling dat u dit veld niet veranderd! <br>
```

Comment:
Vreemde helptekst. Of duidelijk maken wat de gevolgen zijn als de taal wordt aangepast, of dit dient niet mogelijk te zijn.